modules/dashboard/src/components/DebugChannel.js

```diff
@@ -7,56 +7,29 @@ import {
   TextField,
   Typography,
   withStyles,
+  styled,
 } from "@material-ui/core";
 import { Search as SearchIcon } from "@material-ui/icons";
 import PropTypes from "prop-types";
 import { bigNumberify } from "ethers/utils";
+import JSONTree from "react-json-tree";
 
-const styles = {
-  top: {
-    display: "flex",
-    flexWrap: "wrap",
-    flexDirection: "row",
-    width: "100%",
-    height: "100%",
-    justifyContent: "flex-start",
-    alignItems: "center",
-  },
-  card: {
-    marginTop: "15%",
-    display: "flex",
-    height: "320px",
-    width: "320px",
-    alignItems: "center",
-    justifyContent: "center",
-    margin: "0% 2% 0% 2%",
-    border: "3px solid #002868",
-    textDecoration: "none",
-    "&:hover": { backgroundColor: "rgba(0,40,104,0.2)" },
-  },
-  cardText: {
-    textAlign: "left",
-    fontStyle: "italic",
-    fontWeight: "700",
-    fontSize: "24px",
-    color: "#002868",
-    textDecoration: "none",
-  },
-  channelStateGrid: {},
-  icon: {
-    color: "#002868",
-  },
-  xPubEntry: {
-    width: "40%",
-    margin: "2% 5% 5% 1%",
-  },
-  error: {
-    color: "red",
-  },
-  errorWrap: {
-    width: "100%",
-  },
-};
+const RootGrid = styled(Grid)({
+  flexGrow: 1,
+  padding: "5%",
+});
+
```

Comment:
this is *nice*

Reply:
Right, I'm loving this styled-components syntax.